src/pipeline.ml

```diff
@@ -44,7 +44,8 @@ let install_package_archive opam_image =
   copy ~chown:"0:0" ~from:"archive" ~src:["/home/opam/opam-repository/cache"] ~dst:"/cache" ()
 
 (* Generate a Dockerfile to install OCaml compiler [switch] in [opam_image]. *)
-let install_compiler_df ~os_family ~arch ~switch ?windows_port opam_image =
+let install_compiler_df ~arch ~distro ~switch ?windows_port opam_image =
+  let os_family = Dockerfile_distro.os_family_of_distro distro in
```

Comment:
I would put this above `personality` below (i.e. closer to where it's actually used)